stdlib/Sockets/src/IPAddr.jl

```diff
@@ -58,6 +58,8 @@ print(io::IO,ip::IPv4) = print(io,string((ip.host&(0xFF000000))>>24),".",
                                   string((ip.host&(0xFF00))>>8),".",
                                   string(ip.host&0xFF))
 
+zero(::Type{IPv4}) = IPv4(zero(UInt32))
```

Comment:
You probably need
```suggestion
Base.zero(::Type{IPv4}) = IPv4(zero(UInt32))
```
because I suspect `zero` isn't imported into Sockets.